inst/extdata/database/flow_types.csv

```diff
@@ -1,27 +1,358 @@
+hvcs,energycontent_HHV,,,
+hvcs,density_norm,,,
+hvcs,density_std,,,
+urea,name,Urea,,
+urea,default_unit,t,,
+urea,energycontent_LHV,,,
+urea,energycontent_HHV,,,
+urea,density_norm,1.34 g/cm³,,
+urea,density_std,1.34 g/cm³,,
+water,name,Water,,
+water,default_unit,t,,
+water,energycontent_LHV,,,
+water,energycontent_HHV,,,
+water,density_norm,998.21  kg/m³,Lemmon2023,
+water,density_std,999.84 kg/l,Lemmon2023,
+oxygen,name,Oxygen,,
+oxygen,default_unit,t,,
+oxygen,energycontent_LHV,,,
+oxygen,energycontent_HHV,,,
+oxygen,density_norm,1.3312 kg/m³,Lemmon2023,
+oxygen,density_std,1.4290 kg/m³,Lemmon2023,
+nitrogen,name,Nitrogen,,
+nitrogen,default_unit,t,,
+nitrogen,energycontent_LHV,,,
+nitrogen,energycontent_HHV,,,
+nitrogen,density_norm,1.1648 kg/m³,Lemmon2023,
+nitrogen,density_std,1.2505 kg/m³,Lemmon2023,
+graph_electr,name,Graphite electrodes,,
+graph_electr,default_unit,t,,
+graph_electr,energycontent_LHV,,,
+graph_electr,energycontent_HHV,,,
+graph_electr,density_norm,,,
+graph_electr,density_std,,,
+steam,name,Steam,,
+steam,default_unit,MWh,,
+steam,energycontent_LHV,,,
+steam,energycontent_HHV,,,
+steam,density_norm,,,
+steam,density_std,,,
+ft_fuel,name,Fischer-Tropsch diesel,,
+ft_fuel,default_unit,MWh;LHV,,
+ft_fuel,energycontent_LHV,373112.39 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+ft_fuel,energycontent_HHV,392300.51 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+ft_fuel,density_norm,3017 g/gal,GREET2008,"not explicitly given if NTP or STP, assumed that value does not change muchh in this temperature range"
+ft_fuel,density_std,3017 g/gal,GREET2008,"not explicitly given if NTP or STP, assumed that value does not change muchh in this temperature range"
+gasoline,name,Gasoline,,
+gasoline,default_unit,MWh;LHV,,
+gasoline,energycontent_LHV,318131.682 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+gasoline,energycontent_HHV,341511.1742 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+gasoline,density_norm,702.61 kg/m³,Lemmon2023,
+gasoline,density_std,718.61  kg/m³,Lemmon2023,
+crude_oil,name,Crude oil,,
+crude_oil,default_unit,MWh;LHV,,
+crude_oil,energycontent_LHV,415592.35 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+crude_oil,energycontent_HHV,443411.75 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+crude_oil,density_norm,3205 g/gal,GREET2008,"not explicitly given if NTP or STP, assumed that value does not change muchh in this temperature range"
+crude_oil,density_std,3205 g/gal,GREET2008,"not explicitly given if NTP or STP, assumed that value does not change muchh in this temperature range"
+bitumen,name,Bitumen,,
+bitumen,default_unit,MWh;LHV,,
+bitumen,energycontent_LHV,"585055,8263 Btu/kg",GREET2008,calculated from values Energy/volume and density given in the source
```

Comment:
Is this supposed to be comma or perhaps rather a period?

Reply:
indeed it is supposed to be a period